### PR TITLE
updates for Cumulu v18.5.1

### PR DESCRIPTION
**Pull request by lindsleycj**

### CODE REVIEW

**Review by mikedorfman — APPROVED** [review]

**Review by reweeden — COMMENTED** [review]
Should we add these variables to our RDS module? https://github.com/nasa/cumulus/blob/v18.5.1/tf-modules/cumulus-rds-tf/variables.tf#L178-L188

File: rds/variables.tf [hunk 1]
```diff
@@ -77,3 +77,15 @@ variable "snapshot_identifier" {
...
+  description = "If set to true, disable use of SSL with Core database connections."
+  type        = bool
+  default     = true
+}
+
+variable "rejectUnauthorized" {
+  description = "If disableSSL is false or not set, set to false to allow self-signed certificates or non-supported CAs."
+  type        = bool
+  default     = false
+}
```
Comment (reweeden) [inline rds/variables.tf:91]: I've been trying to keep the cumulus variables in this file in alphabetical order.

**Review by reweeden — COMMENTED** [review]
I'd like to deploy this to our dev stack now that we have an Action set up for that. I'll use the cirrus-core PR branch for now since there isn't a release yet.

File: rds/variables.tf (hunk 1 above)
Comment (lindsleycj) [reply rds/variables.tf]: Sounds good.  Have reordered them.

Comment (lindsleycj) [reply rds/variables.tf]: Made the same change to the variables in main.tf

**Review by reweeden — APPROVED** [review]
Good call! The deploy succeeded so I think we're good!

**Review by gjclark — APPROVED** [review]

**Pull request merged**